contrib/completion/zsh/_docker-compose

```diff
@@ -19,52 +19,49 @@
 #  * @felixr docker zsh completion script : https://github.com/felixr/docker-zsh-completion
 # -------------------------------------------------------------------------
 
-# For compatibility reasons, Compose and therefore its completion supports several
-# stack compositon files as listed here, in descending priority.
-# Support for these filenames might be dropped in some future version.
-__docker-compose_compose_file() {
-    local file
-    for file in docker-compose.y{,a}ml ; do
-        [ -e $file ] && {
-            echo $file
-            return
-        }
-    done
-    echo docker-compose.yml
+__docker-compose_q() {
+    docker-compose 2>/dev/null $compose_options "$@"
 }
 
 # Extracts all service names from docker-compose.yml.
```

Comment:
@aeriksson The comment refers to the previous way of harvesting the service names from docker-compose.yml. The comment pointed out that the service list had some limitations: it did not take into account several other configuration files that were added over time.

Now, we use an official API for that task: `docker-compose config`. We let `docker-compose` combine all the configuration sources. But the comment was not updated and now is wrong.

There are two ways to resolve this mismatch: 

1. change the comment to "Extracts all service names from the output of docker-compose config"
2. drop the comment

I favour solution 2 because the comment from solution 1 is trivial. The new code does not need a comment any more.